migration/migrator/migrations/system/20191026164637_ducky.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for the Submitty system."""
+import os
+import json
+from pathlib import Path
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+
+    tgt = str(Path(config.submitty['submitty_install_dir'], 'config', 'submitty.json'))
+
+    with open(tgt, 'r') as infile:
+        data = json.load(infile)
+
+    data['enable_banner'] = False
```

Comment:
Given this is hidden in a config file for all of submitty I feel like "enable_banner" is not very clear. I think it should be something like "enable_duck_banner" or something that says exactly what banner this is.

Also this is making a change for a migration but I assume there is a default config somewhere that you should also add this to so when somebody makes a new install it is in the default config file.